README.md

```diff
@@ -2,37 +2,39 @@
+* Render the list of stores from the stores.json file in alphabetical order using a template.
+* Use postcodes.io to get the latitude and longitude for each postcode. Render them next to each store location in the template.
+* Build the functionality that allows you to return a list of stores in a given radius of a given postcode in the UK. The list must be ordered from north to south. No need to render anything, but the function needs to be unit tested.
 
-### If you're applying for a full stack role
-
-* Build an API that returns stores from the `stores.json` file, based on a given search string and unit test it, i.e. return "Newhaven" when searching for "hav" - make sure the search allows to use both city name and postcode
-* Order the results by matching postcode first and then matching city names, i.e. "br" would have "Orpington" as the 1st result (as its postcode is "BR5 3RP"), and then "Bracknell", "Broadstairs", "Tunbridge_Wells", and "Brentford"
-* Using your favourite frontend framework (we'd prefer Vue) on the user-facing side
-* Build a frontend that renders a text field for the query and the list of stores that match it
-* Add suggestions to the query field as you type, with a debounce effect of 100ms and a minimum of 2 characters
-* Limit the results to 3 and lazy load the rest on page scroll
+### If you are applying for a full stack role
+* Build an API that returns stores from the `stores.json` file, based on a given search string and unit test it. For example, return "Newhaven" when searching for "hav". Make sure the search allows to use both city name and postcode.
+* Order the results by matching postcode first and then matching city names. For example, "br" would have "Orpington" as the 1st result as its postcode is "BR5 3RP". Next would be "Bracknell", "Broadstairs", "Tunbridge_Wells", and "Brentford"
+* Using your favourite frontend framework (we would prefer Vue) on the user-facing side:
+  * Build a frontend that renders a text field for the query and the list of stores that match it
+  * Add suggestions to the query field as you type, with a debounce effect of 100ms and a minimum of 2 characters
+  * Limit the results to 3 and lazy load the rest on page scroll
 
 ### Finally
 
-* Zip your code up and upload it into Greenhouse (our recruitment system) using the link provided at the bottom of the email you received
-* Tell us which test you completed (backend or full-stack)
-* Tell us what you would have done differently if you'd have had more time?
-* What bits did you find the toughest? What bit are you most proud of? In both cases, why?
-* What's one thing we could do to improve this test?
+* Zip your code up and upload it to Greenhouse (our recruitment system). Use the link provided at the bottom of the email you received.
+* Provide answers for the following questions with your submission:
+
+1. Which test did you complete? (backend or full-stack)
+2. What you would have done differently if you had more time?
```

Comment:
- `you` and `would` should be swapped around here.
- Using the past continuous tense is weird here. I'd keep it `"if you'd have had more time"`, because it sounds more retrospective.

Reply:
Good spot, thank you! I’m not sure I agree with the second point but only to avoid a contraction and repetition. `”if you would have had more time”` is quite a mouthful and repeats `would` from earlier in the sentence.

Reply:
`If you had chosen to spend more time on this test, what would have done differently?`

Reply:
@NeilCrosby `you have done differently?`

Reply:
@jarryd-gerber Edited. Thanks for those eagle eyes!

Reply:
Thanks for that, I've updated the question to match what @NeilCrosby suggested.